framework/source/class/qx/ui/core/scroll/AbstractScrollArea.js

```diff
@@ -75,6 +75,12 @@ qx.Class.define("qx.ui.core.scroll.AbstractScrollArea",
       this._setLayout(grid);
     }
 
+    // since the scroll container disregards the min size of the scrollbars
+    // we have to set the min size of the scroll area to ensure that the
+    // scrollbars always have an usable size.
+    var size = qx.ui.core.scroll.AbstractScrollArea.DEFAULT_SCROLLBAR_WIDTH;
+    this.set({minHeight: size, minWidth: size});
+
```

Comment:
Following the logic in the comment above, the use of this is to initialize scroll area with minimal dimensions. It seems that we need place for both a horizontal and a vertical scrollbar. This is minimum `2 * qx.ui.core.scroll.AbstractScrollArea.DEFAULT_SCROLLBAR_WIDTH`.

But I doubt that this is enough as the text states that we should have a USABLE size which shows the up/down or left/right button of the scrollbar and maybe a little room for the scroll knob itself.

With having `qx.ui.core.scroll.AbstractScrollArea.DEFAULT_SCROLLBAR_WIDTH` set to 14, the former minWidth of 50 is not so bad (14 *2 + 22 pixel to scroll).

And a second point is that this change introduces `minHeight` and `minWidth` for all themes deriving from `qx.theme.simple.Appearance` which did not define `minHeight` and `minWidht` through the `scrollarea` appearance.